main.py

```diff
@@ -129,6 +129,10 @@ def username_text(text):
         bot.send_message(message.chat.id, username_text("again? 🙄"))
     elif 'exam' in message.text.lower():
         bot.send_message(message.chat.id, username_text("ʕっ•ᴥ•ʔっ You got this!! 🐝🎶"))
+    elif 'joke' in message.text.lower():
+        bot.send_message(message.chat.id, username_text("Jokes in cat . . ."))
+    elif 'animal' in message.text.lower():
+        bot.send_message(message.chat.id, username_text("i have cat and her name is ayang cantik hahahha :)")) 

```

Comment:
Thank you so much @yasminehh19 !, We appreciate your contribution and look forward to working with you more in the future! 

Now, as a fun suggestion, why not try adding specific trigger responses to your commits? For example, I love my cat Ayang Cantik, and it would be hilarious if the code responded with something like "meow" or "purr" every time she's mentioned.

But of course, we don't want to start any fights among our feline friends. We all know how territorial cats can be! So for now, let's limit our trigger responses to animals in general. Who knows, maybe in future updates we can add a function that allows everyone to have their own space for their beloved pets.

Thanks again for your contribution, and happy coding!